### PR TITLE
RB-4 Create apiToken for org and for customers

### PR DESCRIPTION
**Pull request by stymoshchuk**

### CODE REVIEW

File: lib/unit/models/api_token/customer/create_customer_token_request.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Request to create a customer token
+# @see https://docs.unit.co/customer-api-tokens#customers-create-customer-bearer-token
+module Unit
+  module ApiToken
+    module Customer
+      class CreateCustomerTokenRequest
+        attr_reader :customer_id, :scope, :verification_token, :verification_code, :expires_in, :resources
+
+        # @param customer_id [String]
+        # @param scope [String]
```
Comment (AsafPolakovski) [inline lib/unit/models/api_token/customer/create_customer_token_request.rb:12]: this should be an array of strings and we create a string with spaces as separated, like includes but with spaces.

File: lib/unit/models/api_token/customer/create_customer_token_request.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        # @param verification_token [String]
```
Comment (AsafPolakovski) [inline lib/unit/models/api_token/customer/create_customer_token_request.rb:13]: this is optional

File: lib/unit/models/api_token/customer/create_customer_token_request.rb (hunk 1 above)
Comment (stymoshchuk) [reply lib/unit/models/api_token/customer/create_customer_token_request.rb:12]: "scope": "customers accounts" not array

File: lib/unit/models/api_token/customer/create_customer_token_verification.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+# @see https://docs.unit.co/customer-api-tokens#customers-customer-token-verification
+module Unit
+  module ApiToken
+    module Customer
+      class CreateCustomerTokenVerification
+        attr_reader :customer_id, :channel, :phone, :app_hash, :language
+
+        # @param customer_id [String]
+        # @param channel [String]
+        # @param phone [Phone] - optional
```
Comment (AsafPolakovski) [inline lib/unit/models/api_token/customer/create_customer_token_verification.rb:13]: Didnt we make a Phone type?

File: lib/unit/models/api_token/org/create_api_token_request.rb [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+# @see https://docs.unit.co/org-api-tokens#create-org-api-token
+module Unit
+  module ApiToken
+    module Org
+      class CreateApiTokenRequest
+        attr_reader :user_id, :description, :scope, :expiration, :source_ip, :resources
+
+        # @param user_id [String]
+        # @param description [String]
+        # @param scope [String]
```
Comment (AsafPolakovski) [inline lib/unit/models/api_token/org/create_api_token_request.rb:13]: see other comment on scope

File: lib/unit/models/api_token/customer/create_customer_token_verification.rb (hunk 3 above)
Comment (stymoshchuk) [reply lib/unit/models/api_token/customer/create_customer_token_verification.rb:13]: soo I specified our type

**Review by AsafPolakovski — CHANGES_REQUESTED** [review]
see comments :)

File: lib/unit/models/api_token/customer/create_customer_token_verification.rb (hunk 3 above)
Comment (AsafPolakovski) [reply lib/unit/models/api_token/customer/create_customer_token_verification.rb:13]: Sorry for some reason I read String

**Review by AsafPolakovski — APPROVED** [review]

**Pull request merged**